### PR TITLE
fix: id:46102 Slider Range Components

### PR DESCRIPTION
**Pull request by CaptainSadaf**
Basic Slider Component Implementation is done, same component will be used in numeric facet widget

### CODE REVIEW

**Comment by emgageadmin** [comment]
https://atera.tpondemand.com/entity/46102

File: src/components/RangeSlider/components/Labelled.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+  const helpTextMarkup = helpText ? (
+    <div  id={helpTextID(id)}>
+      {helpText}
+    </div>
+  ) : null;
+
+  const errorMarkup = error && typeof error !== 'boolean' && (
+    <div >
+      {/* <InlineError message={error} fieldID={id} /> */}
```
Comment (ankitsaraogi) [inline src/components/RangeSlider/components/Labelled.tsx:58]: Do we need this commented code?

**Comment by ankitsaraogi** [comment]
@SadafAdrees : Can you please check the comment & resolve conflict

File: src/components/RangeSlider/components/Labelled.tsx (hunk 1 above)
Comment (CaptainSadaf) [reply src/components/RangeSlider/components/Labelled.tsx]: Yes, we need this commented code, It is not commented now.

**Pull request merged**